### ISSUE TITLE
props and context in createDevTools.js on Next Branch (3.0.0-beta-3)

### ISSUE THREAD

**Issue by caljrimmer** [gaearon/redux-devtools#176]
I am upgrading my isomorphic/universal rendering example to react 0.14.
Branch here : https://github.com/caljrimmer/isomorphic-redux-app/tree/react-upgrade-0.14

With 3.0.0-beta-3, I have an issue with the context in lib/createDevTools.js because of the way I am attaching the DevTools panel.

In my branch...

```js
//In server/createDevToolsWindow.js

import DevTools from './server/devtools'

export default function createDevToolsWindow(store) {
  setTimeout(() => ReactDOM.render(
    //Passing the store in as a prop
    <DevTools store={store} />,
    win.document.body.appendChild(document.createElement('div'))
  ), 10);
}
```
```js
//In server/devtools.js
export default DevTools(
  <DockMonitor defaultSize={1} toggleVisibilityKey='H' changePositionKey='P'>
    <LogMonitor />
  </DockMonitor>
);
```

I prefer to open the panel in a separate window as this removes isometric rendering inconsistency errors. I am not providing the state as part of the context but as a prop. I am not sure if I am using an anti-pattern here? 

I have had to amend the redux-devtools/createDevTools.js to make this work by...

```js
//In src/createDevTools.js

return class DevTools extends Component {
    static contextTypes = {
      store: PropTypes.object
    };

    static propTypes = {
      store: PropTypes.object
    };

    static instrument = () => enhancer;

    constructor(props, context) {
      super(props, context);
      if(context.store){
          this.liftedStore = context.store.liftedStore;
      }else{
          this.liftedStore = props.store.liftedStore;
      }
    }

    render() {
      return (
        <ConnectedMonitor {...monitorProps}
                          store={this.liftedStore} />
      );
    }
  };
```

I can folk and PR if you think it is correct way to go about this. Or am I going about this the wrong way?

**Comment by gaearon** [comment]
Why not

```js
export default function createDevToolsWindow(store) {
  setTimeout(() => ReactDOM.render(
    <Provider store={store}>
      <DevTools />
    </Provider>,
    win.document.body.appendChild(document.createElement('div'))
  ), 10);
}
```

?

Although we should probably support passing `store` prop too.

**Comment by gaearon** [comment]
I'm happy to accept a PR fixing this.

**Comment by caljrimmer** [comment]
Of course. I couldn't see the wood for the trees. Daft.

I will PR it this weekend anyway.

**Comment by gaearon** [comment]
Fixed in #178.
